tests/js/destructuring/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -126,8 +126,11 @@ printWidth: 80
 const { foo, bar: bazAndSomething, quxIsLong } = someBigFunctionName("foo")("bar");
 
 =====================================output=====================================
-const { foo, bar: bazAndSomething, quxIsLong } =
-  someBigFunctionName("foo")("bar");
+const {
+  foo,
+  bar: bazAndSomething,
+  quxIsLong,
+} = someBigFunctionName("foo")("bar");
```

Comment:
Similar case changed by this PR but I think **not** by PR #10222.

- [playground of this case in Prettier 2.2.1](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEwA6VOYBmEEANHgQEYCGATkpjQF4CCUAJgMoQC2cMACwCWUAObl8mAI4BXAB4BJNABlo4vAF9MAXkxpecAEJDRAMRlQwMIdABy1PgAocIYhBcBKZyBq1PAbhBSEAgAB2t0ZFA6WggAdwAFOgQ0ZBBqABs46gBPVODKWmowAGt+TlDikVFkGFoZOGC4Hko4dnY25WoxGWpROFMIWh5qGGsxNOoZGAggkAEYHgyAdWF4NEqwOE4UoWsANz2ctLA0fJARNDhaGASi0RHkQkyr4IArNDlDItLyhzhlCI4E8Xo0QB85Jxqhk4ABFGQQeAgjKvECVWhXWhpGitDJzUK0EQwZZCdiCZAADgADMECRArssiqE0gS4Jj9sDgrJEXA7mFUihqGgALRQOBtNpzWhwWRCaV3PqPJDPFFgq48IS1epq6FwhFI5Wg4IwaiUElkgTIABMxqKQgy1QAwrwlSA2QBWOYyK4AFVNApVqP2DQUHAQME4YEJ4TYXBgORhyKuGg0QA)
- [playground of this case on PR 10222](https://deploy-preview-10222--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuc0DOMAEwA6VOYBmEEANHgQEYCGATkpjQF4CCUAJgMoQC2cMACwCWUAObl8mAI4BXAB4BJNABlo4vAF9MAXkxpecAEJDRAMRlQwMIdABy1PgAocIYhBcBKZyBq1PAbhBSEAgAB2t0ZFA6WggAdwAFOgQ0ZBBqABs46gBPVODKWmowAGt+TlDikVFkGFoZOGC4Hko4dnY25WoxGWpROFMIWh5qGGsxNOoZGAggkAEYHgyAdWF4NEqwOE4UoWsANz2ctLA0fJARNDhaGASi0RHkQkyr4IArNDlDItLyhzhlCI4E8Xo0QB85Jxqhk4ABFGQQeAgjKvECVWhXWhpGitDJzUK0EQwZZCdiCZAADgADMECRArssiqE0gS4Jj9sDgrJEXA7mFUihqGgALRQOBtNpzWhwWRCaV3PqPJDPFFgq48IS1epq6FwhFI5Wg4IwaiUElkgTIABMxqKQgy1QAwrwlSA2QBWOYyK4AFVNApVqP2DQUHAQME4YEJ4TYXBgORhyKuGg0QA)
- [playground of this case on recent PR 10416 (should be same as main at this point)](https://deploy-preview-10416--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuc0DOMAEwA6VOYBmEEANHgQEYCGATkpjQF4CCUAJgMoQC2cMACwCWUAObl8mAI4BXAB4BJNABlo4vAF9MAXkxpecAEJDRAMRlQwMIdABy1PgAocIYhBcBKZyBq1PAbhBSEAgAB2t0ZFA6WggAdwAFOgQ0ZBBqABs46gBPVODKWmowAGt+TlDikVFkGFoZOGC4Hko4dnY25WoxGWpROFMIWh5qGGsxNOoZGAggkAEYHgyAdWF4NEqwOE4UoWsANz2ctLA0fJARNDhaGASi0RHkQkyr4IArNDlDItLyhzhlCI4E8Xo0QB85Jxqhk4ABFGQQeAgjKvECVWhXWhpGitDJzUK0EQwZZCdiCZAADgADMECRArssiqE0gS4Jj9sDgrJEXA7mFUihqGgALRQOBtNpzWhwWRCaV3PqPJDPFFgq48IS1epq6FwhFI5Wg4IwaiUElkgTIABMxqKQgy1QAwrwlSA2QBWOYyK4AFVNApVqP2DQUHAQME4YEJ4TYXBgORhyKuGg0QA)
- [playground of this case in this new PR](https://deploy-preview-10643--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuc0DOMAEwA6VOYBmEEANHgQEYCGATkpjQF4CCUAJgMoQC2cMACwCWUAObl8mAI4BXAB4BJNABlo4vAF9MAXkxpecAEJDRAMRlQwMIdABy1PgAocIYhBcBKZyBq1PAbhBSEAgAB2t0ZFA6WggAdwAFOgQ0ZBBqABs46gBPVODKWmowAGt+TlDikVFkGFoZOGC4Hko4dnY25WoxGWpROFMIWh5qGGsxNOoZGAggkAEYHgyAdWF4NEqwOE4UoWsANz2ctLA0fJARNDhaGASi0RHkQkyr4IArNDlDItLyhzhlCI4E8Xo0QB85Jxqhk4ABFGQQeAgjKvECVWhXWhpGitDJzUK0EQwZZCdiCZAADgADMECRArssiqE0gS4Jj9sDgrJEXA7mFUihqGgALRQOBtNpzWhwWRCaV3PqPJDPFFgq48IS1epq6FwhFI5Wg4IwaiUElkgTIABMxqKQgy1QAwrwlSA2QBWOYyK4AFVNApVqP2DQUHAQME4YEJ4TYXBgORhyKuGg0QA)

I would agree with @thorn0 to avoid adding too many cases from #10222 to the CHANGELOG but thought this would be a nice case of something else now fixed (by *this* PR).

Reply:
Think of this PR as of part of #10222, not a thing of its own.

Reply:
Prettier is balancing a bunch of trade-offs. E.g. this specific problem
```js
const { foo, bar: bazAndSomething, quxIsLong } = someBigFunctionName("foo")(
  "bar"
);
```
is the "lone short argument problem" (#3014). Even though for this specific input it's gone, this problem hasn't been fully fixed. That's why showing this example in the changelog would be misleading.

Reply:
Makes sense to me, thanks!